src/main/java/smsk/smoothscroll/cfg/SmScCfg.java

```diff
@@ -36,35 +37,35 @@ public class SmScCfg extends NewConfig {
             "0 % means animation off (no smoothness) and bigger values mean slower animation speed (high smoothness).",
             "Press F3+T in a world to update the config.",
             "To access config ingame, use the mod modmenu."
-        ))).build(),
+        ))).translatable("smoothscroll.config.notes").build(),
         new CfgValueBuilder("Hotbar", new ArrayList<CfgValue>(Arrays.asList(
-            new CfgValueBuilder("Smoothness", 20f).minMax(0, 100).map(0, "Off").map(100, "No Scrolling").step(1).format("%s: %s %%").build(),
-            new CfgValueBuilder("Rollover", true).build()
-        ))).build(),
+            new CfgValueBuilder("Smoothness", 20f).translatable("smoothscroll.config.hotbar.smoothness").minMax(0, 100).map(0, "smoothscroll.config.tooltip.off").map(100, "smoothscroll.config.tooltip.no_scrolling").step(1).format("smoothscroll.config.format.percent").build(),
```

Comment:
tooltip should be reserved for when we want an actual tooltip... i think in this case, the name should be "value" as it is kind of a translation of a value to an alternate name of it